source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -84,15 +84,15 @@ The First Matching Array Element
 
 To update the first array element that matches your query filter, use the
 positional ``$`` operator. The array field must appear as part of your
-query document to use the positional ``$`` operator. 
+query filter to use the positional ``$`` operator. 
 
 Example
 ```````
 
 The following example performs these actions:
 
 - Query for a document with a ``qty`` field containing the value "18"
-- Decrement the first array value that matches the query document by "3"  using the ``inc()`` method of the ``Updates`` builder and the positional ``$`` operator 
+- Decrement the first array value that matches the query filter by "3"  using the ``inc()`` method of the ``Updates`` builder and the positional ``$`` operator 
```

Comment:
This sentence contains too much information which makes it difficult to interpret. Can you think of a way to split the information up such that each sentence only describes one thing about a subject and object? I think we can probably omit mention of the Updates builder to keep it short. E.g.
"If an array value matches the query filter, the write operation performs the update. The update decrements... "

Reply:
I agree, I removed the clutter in each bullet.